src/server/utilities.ts

```diff
@@ -192,6 +192,33 @@ namespace ts.server {
         return <any>arr;
     }
 
+    export function enumerateInsertsAndDeletes<T>(a: SortedReadonlyArray<T>, b: SortedReadonlyArray<T>, inserted: (item: T) => void, deleted: (item: T) => void, compare?: (a: T, b: T) => number) {
+        if (!compare) compare = (a, b) => a > b ? 1 : a < b ? -1 : 0;
+        let aIndex = 0, aLen = a.length;
+        let bIndex = 0, bLen = b.length;

```

Comment:
```
15:47:16 src/server/utilities.ts
15:47:16 ERROR: 197:25  prefer-const    Identifier 'aLen' is never reassigned; use 'const' instead of 'let'.
15:47:16 ERROR: 198:25  prefer-const    Identifier 'bLen' is never reassigned; use 'const' instead of 'let'.
```